packages/sdk/src/engine-commands/queryEngineCommons.ts

```diff
@@ -75,11 +76,24 @@ export function loadNodeAPILibrary(queryEnginePath: string) {
   return pipe(
     E.tryCatch(
       () => load<NodeAPILibraryTypes.Library>(queryEnginePath),
-      (e) => ({
-        type: 'connection-error' as const,
-        reason: 'Unable to establish a connection to query-engine-node-api library',
-        error: e as Error,
-      }),
+      (e) => {
+        const error = e as Error
+        const defaultErrorMessage = `Unable to establish a connection to query-engine-node-api library.`
+        const proposedErrorFixMessage = match(error.message)
+          .when(
+            (errMessage) => errMessage.includes('openssl'),
+            () => {
+              return ` Have you tried installing 'openssl'?`
```

Comment:
I fear that is not a super helpful error message (The snap thing for example would not be really covered by this msg, wouldn't it?). 

Maybe we should ahve a troubleshooting entry in our docs and a pris.ly redirect pointing there that we can link to here instead?

Reply:
We can talk about this later with Lucy. I like the idea

Reply:
If you want to merge this without further changes on that message, extract this suggestion into an issue pls.
Maybe still ask @pantharshit00 for a potentially better description of what there is to do right now to fix this in a general (non snap specific) way.

Reply:
I had sent a Slack message to @pantharshit00 asking him to take a look here, I'll ping him back

Reply:
Harshit is out sick today. 

The wording here is pretty broad in my opinion, and as far as I know it being not installed was never the problem. It was mostly about it not being installed correctly or via the correct method. Maybe we can make it more neutral as "The error message indicates this is a problem with your OpenSSL installation" or something like that.